### PR TITLE
usage of backticks draft

### PR DESCRIPTION
**Pull request by cake17**
draft linked to #1152 after @bcrowe proposal

### CODE REVIEW

File: en/contributing/documentation.rst [hunk 1]
```diff
@@ -72,26 +73,32 @@ preceded and followed by a blank line.
...
 
 Inline Markup
 -------------
 
 * One asterisk: *text* for emphasis (italics)
+  We'll use it for Object names, excluding the following word "object",
+  for method/function names -- include "()".
 
-  * ``*text*``
+  * ``*PagesController*``, ``**config()**``, etc.
```
Comment (markstory) [inline en/contributing/documentation.rst]: I've been trying to use inline code for these things, but I'm flexible if the larger group thinks that italics would be better.

Comment (cake17) [inline en/contributing/documentation.rst]: I can move them to inline code.
What sort of things could be in italics ? Currently, I could find general emphasis, some words or portions of sentences.

Comment (markstory) [inline en/contributing/documentation.rst]: General emphasis I think.

**Comment by cake17** [comment]
Any modification on this ?

**Comment by markstory** [comment]
I think @bcrowe forgot about this :smile:

**Comment by bcrowe** [comment]
Sorry! I was more-so waiting on some feedback with what styles to use where. But, seeing as no one else had any strong opinions against my initial style usage, I'm all in. :+1:

**Comment by cake17** [comment]
ok :) it can still be changed with a PR after if necessary

**Pull request merged**